src/utils.ts

```diff
@@ -55,6 +57,15 @@ export function getRootDirectory(file: File): string {
   return path.normalize(dir).slice(root.length).split(path.sep)[0];
 }
 
+/**
+ * Generates a public and private key from a provided, secure seed.
+ * @param seed - A secure seed.
+ */
+export function keyPairFromSeed(seed: string): { publicKey: PublicKey; privateKey: SecretKey } {
```

Comment:
I would prefix that function name with a verb like `generate` or `create`

Reply:
I think it's fine as-is.